core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -43,7 +48,7 @@ class HttpWorkspaceManagerDAO(apiClientProvider: WorkspaceManagerApiClientProvid
       .managedBy(ManagedBy.USER)
 
   override def getWorkspace(workspaceId: UUID, ctx: RawlsRequestContext): WorkspaceDescription =
-    getWorkspaceApi(ctx).getWorkspace(workspaceId)
+    getWorkspaceApi(ctx).getWorkspace(workspaceId, null)
```

Comment:
WSM's `getWorkspace` added a new parameter `minimumHighestRole` in the updated version; it has a default (`READER`), so I'm passing a `null` for now.